pkgs/development/tools/codespell/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildPythonApplication, fetchPypi, pytest, chardet }:
+buildPythonApplication rec {
+  pname = "codespell";
+  version = "1.15.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0c211rzfgmwls8ab8fj21xp9bhxk6ys3xw8w7chp4arjlifc26wg";
+  };
+
+  checkInputs = [ pytest chardet ];
+  checkPhase = ''
+    # We don't want to be affected by the presence of these
+    rm -r codespell_lib setup.cfg
+    # test_command assumes too much about the execution environment
+    pytest --pyargs codespell_lib.tests -k "not test_command"
+  '';
+
+  meta = {
+    description = "Fix common misspellings in text files";
```

Comment:
Upstream: "check code for common misspellings"

So "text files" doesn't seem quite right to me.

Reply:
You're right, that's better. I chose the first sentence of the upstream README.